modules/account-lib/test/unit/coin/dot/transactionBuilder/base.ts

```diff
@@ -176,56 +176,6 @@ describe('Dot Transfer Builder', () => {
     });
   });
 
-  describe('add signature', () => {
```

Comment:
removing this test as we other tests are implicitly testing this when using `.addSignature` to sign a transaction.